handlers/ec2/resources/scripts/nova-essex-prepare-key

```diff
@@ -56,41 +56,38 @@ try:
     #DO THE WORK                                                                                                                                                                                       
     LOG.debug("nova-essex-prepare-key: " + str(sys.argv))
 
-    if str(os.environ['AMI_NAME']) and str(os.environ['AKI_NAME']) and str(os.environ['ARI_NAME']):
```

Comment:
This is the main change

Reply:
I had inserted this statement to by-pass SSH key insertion for Windows images. It was my presumption that Windows images would have only AMI whereas Linux images would have AMI,AKI,ARI. I will find a better way for Windows images for inserting SSH keys. (Probably install an SSH server to the image.)